jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/sender/PushNotificationSenderEndpoint.java

```diff
@@ -69,10 +69,9 @@
      * </pre>
      *
      * Details about the Message Format can be found HERE!
+     * <p/><p/>
      *
-     * @HTTP 202 (Accepted) Indicates the Job has been accepted and is being process by the AeroGear UnifiedPush Server.
-     * @HTTP 401 (Unauthorized) The request requires authentication.
-     * @RequestHeader aerogear-sender The header to identify the used client. If the header is not present, the standard "user-agent" header is used.
+     * <b>Request Header</b> {@code aerogear-sender} uses to identify the used client. If the header is not present, the standard "user-agent" header is used.
```

Comment:
I will create a ticket in MireDot issue tracker to consider adding a javadoc tag `@requestheader`, cause they support just `@responseheader` tag right now http://www.miredot.com/docs/manual/source-code-enhancements/javadoc/.
I think that this tag could be useful for many cases.